Parser/ABOParser.php

```diff
@@ -15,12 +16,12 @@
  */
 class ABOParser extends Parser
 {
-    const LINE_TYPE_STATEMENT   = 'statement';
+    const LINE_TYPE_STATEMENT = 'statement';
```

Comment:
Why have you changed indentions? IMHO it's faster/better to read when the related ones are sorted the same way.